src/scripts/main.js

```diff
@@ -354,7 +354,40 @@
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+'use strict';
+
+const table = document.querySelector('.dashboard');
+
+function createCell(content) {
+  const cell = document.createElement('td');
+  cell.textContent = content;
+  return cell;
+}
```

Comment:
The `createCell` function is a good example of a helper function. It's small, does one thing, and does it well. However, it could be made more reusable. Currently, it only creates 'td' elements. What if you also wanted to create 'th' elements for table headers? Consider adding another parameter to specify the type of table cell to create.